tests/testthat/test-truncate_linelist.R

```diff
@@ -3,44 +3,51 @@ ll <- sim_linelist()
 
 test_that("truncate_linelist works as expected with defaults", {
   ll_trunc <- truncate_linelist(ll)
-  # in this example dataset only the outcome col is adjusted from truncate_linelist
+  # in this example dataset one case is removed
   expect_false(identical(ll$date_outcome, ll_trunc$date_outcome))
+  expect_gt(nrow(ll), nrow(ll_trunc))
+})
+
+test_that("truncate_linelist works as expected with modified truncation_time", {
+  ll_trunc <- truncate_linelist(
+    linelist = ll,
+    truncation_time = 60
+  )
+  expect_gt(nrow(ll), nrow(ll_trunc))
 })
 
-test_that("truncate_linelist works as expected with modified delay", {
+test_that("truncate_linelist works as expected with different units", {
   ll_trunc <- truncate_linelist(
-    ll,
-    delay = function(x) rlnorm(n = x, meanlog = 3, sdlog = 2)
+    linelist = ll,
+    truncation_time = 3,
```

Comment:
could test with non-integer truncation time to cover a slightly more special case.

Reply:
Test added in 7d63f27acf59ec8962ecd0f2a7054173d9c4f9f3.